recipe/post-link.sh.in

```
@@ -2,13 +2,37 @@
 for file in \
 	bin/grass \
 	opt/grass/etc/python/grass/app/resource_paths.py \
-	opt/grass/etc/python/grass/lib/*.py \
 	opt/grass/etc/fontcap \
 	opt/grass/demolocation/.grassrc85 \
-	opt/grass/docs/html/*.html \
 	; do
-	sed -Ei \
-		-e "s#@compile_prefix@#$PREFIX#g" \
-		-e "s#lib64/grass85#opt/grass#g" \
-		$PREFIX/$file
+	if [ -f "$PREFIX/$file" ]; then
```

Is it just for safety or did you have any issues without this check? I don't see any reason for separating wildcards from this for loop. We know those directories exist and I don't see again a reason for an extra check for directory existence.

---

We know these files and directories exist after a successful build, but if for some reason, any of them doesn't exist, we should fail at this point instead of just skipping non-existing files or directories, causing an incomplete packing. I would just fail if such file doesn't exist whether using `exit 1` or simply through `sed`.

---

Since I’ve run into the same error mentioned in the PR description, I just want to make sure we capture and handle it properly.